### PR TITLE
Cleanup/1.2

### PR DESCRIPTION
**Pull request by Snugug**
Resolves https://github.com/sasstools/sass-lint/issues/123

`DCO 1.1 Signed-off-by: Sam Richard <snugug@gmail.com>`

### CODE REVIEW

**Comment by bgriffith** [comment]
This is great! Should make maintaining the tests a lot easier! 

I've looked through the changes - all good!

File: lib/config/sass-lint.yml [hunk 1]
```diff
@@ -18,23 +18,23 @@ rules:
...
-  no-duplicate-property: 1
+  no-duplicate-properties: 1
   no-empty-rulesets: 1
   no-extends: 0
   no-ids: 1
   no-important: 1
   no-warn: 1
+  no-color-keywords: 1
+  no-invalid-hex: 1
+  no-css-comments: 1
```
Comment (bgriffith) [inline lib/config/sass-lint.yml]: Do we want these to be ordered alphabetically?

Comment (Snugug) [inline lib/config/sass-lint.yml]: As these are our default values and we have a separate [sample config](https://github.com/sasstools/sass-lint/blob/master/docs/sass-lint.yml), I don't really care if it's alphabetical or not.

Comment (bgriffith) [inline lib/config/sass-lint.yml]: Yeah ok - fair point hah

**Pull request merged**